ember.md

```diff
@@ -183,3 +185,25 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 </div>
 
 ```
+<a name="testing"></a>
+## Testing
+
+<a name="testing--test-scripts"></a>
+### 5.1 Test Scripts
+
+> Why? To allow us to establish and hold to a standard of code coverage in all of our apps with meaningful test writing and the ability to gate deployments when those standards are not met.
+
+We use three devDependencies to compile our test scripts.  
+1. `ember-exam` - For allowing more control over which tests run in Ember CLI QUnit
+1. `cross-env` - For allowing single commands for multiple test environments
+1. `ember-cli-code-coverage` - For testing the percentage of code coverage with Istanbul
+
+The `scripts` section in your __package.json__ file should include the following...
+
+``` javascript
+"scripts": {
+    "test": "cross-env COVERAGE=true ember exam",
+    "code-server": "cross-env COVERAGE=true ember exam --server",
+    "test-server": "cross-env COVERAGE=true ember exam --filter=!acceptance --server"
```

Comment:
The `test-server` command should not be using exam to filer out acceptance tests. This will prevent these tests from being ran when testing is checked on the build server

Reply:
That's why there are two, one for `code coverage` and one for `test coverage`. I think only the `code coverage` results should be checked on the build server. We can discuss though.